app/scripts/modules/google/src/serverGroup/configure/serverGroupCommandBuilder.service.js

```diff
@@ -274,6 +274,10 @@ module.exports = angular
           if (command.labels['spinnaker-server-group']) {
             delete command.labels['spinnaker-server-group'];
           }
+
+          if (command.labels['spinnaker-moniker-sequence']) {
```

Comment:
Can you drop a comment here explaining the expected behavior here? It's not too clear why it's important to clear this key.